lib/services/plugins-service.ts

```diff
@@ -88,18 +92,20 @@ export class PluginsService implements IPluginsService {
 				} else {
 					this.$errors.fail("Plugin '%s' is already installed", pluginName);
 				}
+			} else if(installedPlugin && version) {
```

Comment:
I would expect to check the version in case I've specified it and if it is valid for this plugin, just to use the specified version